src/tests/QueryBuilderTests.cpp

```diff
@@ -278,6 +279,28 @@ TEST_CASE_METHOD(SqlTestFixture, "SqlQueryBuilder.FieldsWithBelongsTo", "[SqlQue
 
 }
 
+
+struct QueryBuilderTestEmailWithAliases
+{
+    Field<std::string, SqlRealName { "FAX_ADRESS" }  > email;
+    BelongsTo<&UsersFields::name, SqlRealName{ "UUID"}> user;
```

Comment:
```suggestion
    Field<std::string, SqlRealName { "FAX_ADRESS" }> email;
    BelongsTo<&UsersFields::name, SqlRealName{ "USER_REC" }> user;
```

"UUID" is not the best column name in an SQL table, as some might have it as a keyword. 
But actually, I wanted to suggest proper formatting

Reply:
Updated it, and also clang-format was ok with this formatting xD

Reply:
This IS weird. Interesting!